problemsets-mat223/modules/module6.tex

```diff
@@ -214,9 +223,9 @@
 \end{example}
 
 \begin{example}
-	Let $\ell=\Span\Set*{\mat{1\\2},\mat{-2\\-4}, \mat{1/2\\1}}$.
+	Let $\ell=\Span\Set*{\mat{1\\2},\mat{-2\\-4}, \mat{1/2\\1}}$. Find the dimension of subspace $\ell$.
 
-	XXX Finish
+    Most of the work here comes from finding a basis for $\ell$, which we've already done earlier. After that, finding the dimension of $\ell$ is as easy as counting the number of elements in that basis. Since $\mathcal B_1 = \Set*{\mat{1\\2}}$ contains one element, $\ell$ is one dimensional\footnote{We could have looked at $\mathcal B_2$ or any of the other infinitely many possible bases for $\ell$ and the result would have been the same - since each of those bases contains only one element.}.
```

Comment:
See earlier about breaking this up into multiple lines. You can be a little more terse. `This is the same $\ell$ as the previous example where we found that \[\mathcal B_1=  .... \qquad\text{and}\qquad \mathcal B_2=...\] were both bases for $\ell$. Therefore ...`